src/test/appVersions.test.js

```diff
@@ -208,7 +208,7 @@ describe("App Version List component", function () {
 describe("App Version component", function () {
 
   beforeEach(function () {
-    var model = {
+    this.model = {
```

Comment:
how about a `let model` ? I might be missing something here, but I don't see the need for a `this.model` ?